core/shared/src/main/scala/it/unibo/scafi/core/Semantics.scala

```diff
@@ -268,7 +268,7 @@ trait Semantics extends Core with Language {
             .toList
       }
 
-    override def elicitAggregateFunctionTag(): Any = Thread.currentThread().getStackTrace()(PlatformDependentConstants.StackTracePosition)
+    override def elicitAggregateFunctionTag(): Any = new Throwable().getStackTrace()(PlatformDependentConstants.StackTracePosition)
```

Comment:
It seems reasonable to be. @metaphori what do you think? Scala native do not support Thread.currentThread.getStackTrace() currently. I tried some micro benchmarking on the performance, and the new Throwable seems to have the same performance as Thread.getStackTrace().

Reply:
I wouldn't change this in `shared`. Isn't it possible to change it specifically for Scala native only?

Reply:
Fair enough. @KevinManciniHull can you check if it is possible?

Reply:
Sorry I looked it quickly. This is fine if it is has no impact on Scala/Java versions and its equivalent also wrt performance